spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
EnqueueJob takes neither _realm_ nor _continent_ parameters. This is trying to clarify that there's an implicit continent parameter (as there is with essentially everything in the spec, but most of the time it is not as relevant). In particular, I had first written the HTML spec calling out to EnqueueJob("ScriptJobs", ScriptEvaluationJob, « _sourceText_ ») but the results were very confusing since it wasn't clear that there was actually one "ScriptJobs" queue per continent, i.e. it wasn't clear what relative order needed to be preserved between.

Note that executing code does _not_ in fact require a realm.